packages/features/data-table/components/filters/MultiSelectFilterOptions.tsx

```diff
@@ -32,31 +32,42 @@ export function MultiSelectFilterOptions({ column }: MultiSelectFilterOptionsPro
       <CommandInput placeholder={t("search")} />
```

Comment:
This change enables "Section" (in both MultiSelectFilterOptions and SingleSelectFilterOptions)

![Screenshot 2025-02-21 at 16 34 37](https://github.com/user-attachments/assets/b533d3da-2814-4a51-a900-1791c95a864d)